lib/engine.py

```diff
@@ -169,8 +178,12 @@ def auto_create_table(self, table, url=None, filename=None, pk=None):
 
             self.auto_get_datatypes(pk, lines, columns, column_values)
 
-        if self.table.columns[-1][1][0][:3] == "ct-" and hasattr(self.table, "ct_names") and not self.table.ct_column in [c[0] for c in self.table.columns]:
-            self.table.columns = self.table.columns[:-1] + [(self.table.ct_column, ("char", 20))] + [self.table.columns[-1]]
+        if self.table.columns[-1][1][0][:3] == "ct-" and hasattr(self.table,
+                                                                 "ct_names") and self.table.ct_column not in [c[0] for c
+                                                                                                              in
+                                                                                                              self.table.columns]:
+            self.table.columns = self.table.columns[
+                :-1] + [(self.table.ct_column, ("char", 20))] + [self.table.columns[-1]]
```

Comment:
This is a confusing split. I'd rather see a `\` after one of the `+`s to break this line.